src/seedu/addressbook/ui/Formatter.java

```diff
@@ -54,6 +55,30 @@ private static String asIndexedList(List<String> listItems) {
         return formatted.toString();
     }
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    public static String getPrintableString(Printable... printables){
```

Comment:
Great job using varargs to allow multiple printables to be passed into the method =).
Do look at the coding standard that there is a ` ` between the method signature and `{`

It should look something like `public static String getPrintableString(...) {`.